tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalJettyBundle.java

```diff
@@ -0,0 +1,144 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.server.core.portal;
+
+import java.net.MalformedURLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.jdt.launching.IVMInstall;
+
+/**
+ * @author Simon Jiang
+ */
+public class PortalJettyBundle extends AbstractPortalBundle  implements PortalBundle
+{
+
+    public static final int DEFAULT_JMX_PORT = 2099;
+
+    public PortalJettyBundle( IPath path )
+    {
+       super(path);
+    }
+
+    protected int getDefaultJMXRemotePort()
+    {
+        return DEFAULT_JMX_PORT;
+    }
+
+    public String getStartMainClass()
+    {
+        return "org.eclipse.jetty.start.Main";
+    }
+
+    public String getStopMainClass()
+    {
+        return "org.eclipse.jetty.start.Main";
+    }
+
+    
+    protected IPath getPortalDir( IPath appServerDir )
+    {
+        IPath retval = null;
+
+        if( appServerDir != null )
+        {
+            retval = appServerDir.append( "/webapps/root" );
+        }
+
+        return retval;
+    }
+
+    protected IPath getPortaGlobalLib()
+    {
+        return new Path(this.bundlePath + "/lib/ext/liferay" );
+    }
+    
+    public IPath[] getRuntimeClasspath(IVMInstall vmInstall)
```

Comment:
Actually the RuntimeClasspath is no longer going to be needed at all.  This was used to create the runtime classpath container but that is going away in 3.0.  So we can remove it from here and all other places as well.